src/components/Card/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { CardType } from '../Form'
+import CardExpire from './CardExpire'
+import CardNumber from './CardNumber'
+import CardOwner from './CardOnwer'
+import Styled from './index.style'
+
+export interface CardProp {
+  type?: CardType
+  number1: string
+  number2: string
+  number3: string
+  number4: string
+  owner: string
+  month: string
+  year: string
+  size?: 'big' | 'small'
+  setModalIsOpen?: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+const CardTop = React.memo(
+  ({ type }: { type?: CardType }) => (
+    <Styled.CardTop>
+      <Styled.CardText>{type || '클린'}카드</Styled.CardText>
+    </Styled.CardTop>
+  ),
+  (prev, cur) => prev.type === cur.type
+)
+
+const CardMiddle = React.memo(
+  () => (
+    <Styled.CardMiddle>
+      <Styled.CardSmallChip />
+    </Styled.CardMiddle>
+  ),
+  () => true
+)
+
+const Card = ({
+  size = 'small',
+  type,
+  number1,
+  number2,
+  number3,
+  number4,
+  month,
+  owner,
+  year,
+  setModalIsOpen,
+}: CardProp) => (
+  <Styled.Card
+    {...(type && { bgColor: BackgroundColorAccordingToStartsWith[type] })}
+    {...(setModalIsOpen && { ableToModalOpen: true })}
+    {...(setModalIsOpen && { onClick: () => setModalIsOpen(true) })}
+    size={size}
+  >
+    <CardTop type={type} />
+    <CardMiddle />
+    <Styled.CardBottom>
+      <CardNumber
+        number1={number1}
+        number2={number2}
+        number3={number3}
+        number4={number4}
+      />
```

Comment:
```suggestion
      <CardNumber
        {...props}       
      />
```
타입스크립트를 쓰고 있고 받은 props를 그대로 넘겨주는 상황이니 전개 연산자를 활용해도 좋을것 같네요